extensions/wikia/AssetsManager/AssetsManagerController.class.php

```diff
@@ -81,7 +81,11 @@ public function getMultiTypePackage() {
 					$builder = $this->getBuilder( 'sass', $styleFile );
 
 					if ( !is_null( $builder ) ) {
-						 $data .= $builder->getContent();
+						$skin = RequestContext::getMain()->getSkin();
+						if ( $this->app->checkSkin( 'oasis', $skin ) ) {
```

Comment:
You can skip the `$skin` parameter here as it defaults to the skin from `RequestContext`.